CHANGELOG.md

```diff
@@ -1,4 +1,60 @@
 # Change Log
 
+## 3.0.0 (March 31, 2021)
+
+* Adds support for Rest API versioning
+* Remove `ramda` dependency
+* Remove `string` dependency
+* Remove `@procore/js-sdk-endpoints` dependency
+
+### Upgrading
+
+As of v3.0.0, this npm package now defaults to making requests against Procore's new
+`/rest/v1.0` namespace, instead of the now deprecated `/vapid` namespace.
+
+Previous Version Usage
+
+```javascript
+// In Previous Version
+const procore = client(authorizer);
+const { body } = await procore.get({ base: '/vapid/me' });
+// app.procore.com/vapid/me
+```
+
+v3.0.0 Version Usage
+
+```javascript
+const procore = client(authorizer);
+
+// Default version is v1.0
+const { body } = await procore.get({ base: '/me' });
+// app.procore.com/rest/v1.0/me
+
+// Override default version
+const { body } = await procore.get({ base: '/me', version: 'v1.0' });
```

Comment:
```suggestion
const { body } = await procore.get({ base: '/me', version: 'v1.1' });
```

Reply:
I changed this to v1.0 since we don't have a v1.1 version of /me. This endpoint is not the best example. Let me see if I can find a better example.